peer/ping_manager.go

```diff
@@ -196,12 +196,10 @@ func (m *PingManager) pingHandler() {
 	}
 }
 
-// Stop interrupts the goroutines that the PingManager owns. Can only be called
-// when the PingManager is running.
-func (m *PingManager) Stop() error {
+// Stop interrupts the goroutines that the PingManager owns.
+func (m *PingManager) Stop() {
 	if m.pingTicker == nil {
```

Comment:
q: Do we still need ths conditional here? Would it ever be hit?

Reply:
It turns out, yes. Testing this code reveals that if there is a timeout during the init process of the Brontide, then we call `Disconnect` before the ping manager is ever started. Since `Start` is what initializes the `Timer` this will be nil at that point.